SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+Please refer to our [vulnerability disclosure dedicated page](https://www.talend.com/security/vulnerability-disclosure/) first.
+
+## Supported Versions
+
+We use `changeset` to provide versioning for our packages with fixes, features and major release in case of breaking changes. The latest package is always the better one.
+
+We also provide [talend-scripts upgrade:deps](./tools/upgrade-deps) scripts to make it easier to keep our packages and apps aligned and updated.
+
+The script comes with options about security use cases, so please read the documentation first.
+
+Distance to latest from npm is added on a [dependencies.txt](./dependencies.txt) file on each upgrade PR.
+
+## Reporting a Vulnerability
+
+Please open an issue on this github repository if you are outside Talend, or in the company's tracker for your team if you are inside Talend.
```

Comment:
Could this section be changed to instead reference our vulnerability disclosure page?